### DISCUSSION TITLE
global -> local reason in util

### DISCUSSION THREAD

**Discussion by Akrugerus** [Rapptz/discord.py#8117] [Q&A] [answered]
I found @Gobot1234 to blame for [this](https://github.com/Rapptz/discord.py/blob/75d57ab3cee640a5960d4a9c78ee5003c14d11a1/discord/utils.py#L441) when I went to borrow the get and find utility functions from this project. I'm assuming storing builtins as locals is some sort of performance optimization but I'm not familiar enough with Python's internals to understand. Can anyone shed some light for me?

**Comment by Gobot1234** [comment, marked as answer]
I didn't write this, it was there before, I just preserved it when I added support for async iteration for get, it just adds the variables to locals which speedups lookups for tight iterations.

**Reply by Akrugerus** [reply]
That's what I figured, thanks.
